### ISSUE TITLE
Request to deactivate all auto-created “portal rooms” for W3C IRC network

### ISSUE THREAD

**Issue by sideshowbarker** [matrix-org/matrix-appservice-irc#1647]
The W3C would like for all auto-created “portal rooms” for W3C IRC network to be deactivated.

To be clear, we mean the rooms with aliases of the form `#_w3c_#privacy:matrix.org`.

We now are instead using other “plumbed rooms” — native Matrix rooms that we manually bridged to the W3C IRC network.

Those “plumbed rooms” — the ones we created manually and want to keep — all instead have room aliases of the form `#w3c_privacy:matrix.org`

**Comment by Half-Shot** [comment]
We've submitted a change to the configuration, this should apply shortly.

**Comment by sideshowbarker** [comment]
> We've submitted a change to the configuration, this should apply shortly.

Excellent, thanks much — I’ll watch and see

**Comment by sideshowbarker** [comment]
We’re finding that we can still discover and join, e.g., `#_w3c_#privacy:matrix.org` — even though it’s been a couple hours since the configuration change was submitted. We need to wait longer?

**Comment by Half-Shot** [comment]
Yep, It needs me to press a button, will ping when done.

**Comment by Half-Shot** [comment]
Applied.

**Comment by sideshowbarker** [comment]
Howdy~ as I understand it, the change that was made in response to this had the effect of preventing any new rooms from being auto-created — which is great — but the other part we were looking for here is to have all the _already-existing_ rooms to be disabled/removed.

To be clear, we mean all of the existing rooms with room aliases of the form `#_w3c_#privacy:matrix.org` — or to put it in other terms: all rooms that currently show up in the **Explore Rooms** UI in Element, when your server set to the `matrix.org` server, and from the submenu where it lists **Matrix**, **OFTC**, **Snoonet**, and then **W3C**, you select **W3C**:

![image](https://user-images.githubusercontent.com/194984/203910688-66bd25ca-cf8b-42b9-88eb-3e3863a0149c.png)

It’s about 40 rooms in all.

If y’all feel we could be trusted with the ability to control/impersonate the `@w3c-irc:matrix.org` account, then I can imagine one way to handle this is that I could use that myself directly to remove the rooms we want removed.

I’d be willing to agree ahead of time that any mistakes I made were our own responsibility — but, as far as I understand, that `@w3c-irc:matrix.org` account anyway only has Admin access to room/channels that are W3C-owned (either indirectly, by being portal rooms bridged to W3C-owned IRC channels, or else native Matrix plumbed rooms that we created ourselves), then any mistaken deletions would only be affecting our own rooms/channels.

But if it’s preferable not to give over control of the `@w3c-irc:matrix.org` account, then I’d also be happy to do through the rooms one-by-one with somebody who does have control, and explicitly make the sign off on deleting each room.

And also for the record here about why we want to remove those portal rooms: We now have native plumbed rooms for all of them — so the portal rooms are now redundant with our native plumbed rooms — and we would prefer that users now use the plumbed rooms rather than the portal rooms.

And among the reasons why we prefer that is, we have ownership/Admin control over those native plumbed rooms — including spam-control/moderation capabilities — which we do not have with the portal rooms. And one specific reason that’s important is that we’ve had problems with spammers who seem to be coming in by way of the portal rooms, and so we’d like cut off that additional spam vector.
